### ISSUE TITLE
Objects with the same name in the same package

### ISSUE THREAD

**Issue by andreoss** [objectionary/eo#499]
`duplicated-names` does not check across files within a package. 

```
# File: my-test-1.eo
# Erased during compilation`
[]` > my-test
  eq. > @
    1
    0

# File: my-test-2.eo
# Got compiled
[] > my-test
  eq. > @
    1
    1
```

Problem in the current code:
https://github.com/cqfn/eo/blob/master/eo-runtime/src/test/eo/org/eolang/float-tests.eo#L206
https://github.com/cqfn/eo/blob/master/eo-runtime/src/test/eo/org/eolang/int-tests.eo#L203

**Comment by yegor256** [comment]
@andreoss definitely a bug. I think we need an additional XSL checker for this type of error.

**Comment by yegor256** [comment]
@andreoss please, fix this

**Comment by yegor256** [comment]
@andreoss is this fixed? see #1050

**Comment by andreoss** [comment]
@yegor256 Yes, it can be closed now
